README.md

```diff
@@ -59,7 +59,7 @@ python scraper.py
 You can set these env variables:
 
 - `LOG_LVL`: Integer severity value for the loguru library (see [this table](https://loguru.readthedocs.io/en/stable/api/logger.html#levels)). Defaults to 20 (INFO).
-- `LATEST_ONLY`: Set to `1` to scrape only the latest episode of each show defined in `config.yml`. This mode is used for automatically scraping new episode with github actions. Default mode is to scrape all episodes and all data.
+- `LATEST_ONLY`: Set to `1` to scrape only the latest episode of each show defined in `config.yml`. This mode is used for automatically scraping new episode with GitHub actions. Default mode is to scrape all episodes and all data.
```

Comment:
Noticed while looking at your PR, it says set to 1 (which _technically_ works), but in the [JB GH action](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/7af9556773563c205924cf9077757e989527fb4c/.github/workflows/scrape.yml#L40) it's actually using `true`. I think this would be a better reflection in the docs since that's what we're using in prod.

```suggestion
- `LATEST_ONLY`: Set to `true` to scrape only the latest episode of each show defined in `config.yml`. This mode is used for automatically scraping new episode with GitHub actions. Default mode is to scrape all episodes and all data.
```